modules/cpmso-activating.adoc

```diff
@@ -6,4 +6,22 @@
 [id="cpmso-activating_{context}"]
 = Activating the Control Plane Machine Set Operator
 
-To use the Control Plane Machine Set Operator on AWS clusters that are upgraded to version 4.12 or later from an earlier version, you must activate the Operator.
\ No newline at end of file
+To use the Control Plane Machine Set Operator, you must ensure that a `ControlPlaneMachineSet` custom resource (CR) with the correct settings for your cluster exists. On a cluster with a generated CR, you must verify that the configuration in the CR is correct for your cluster and activate the Operator.
+
+.Procedure
+
+. View the configuration of the Operator by running the following command:
++
+[source,terminal]
+----
+$ oc --namespace openshift-machine-api edit controlplanemachineset.machine.openshift.io cluster
+----
+
+. Change the values of any fields that are incorrect for your cluster configuration.
++
+[NOTE]
+====
+For more information about the parameters in the CR, see "Control Plane Machine Set Operator configuration".
+====
+
+. When the configuration is correct, activate the Operator by setting the `.spec.state` field to `Active` and saving your changes.
```

Comment:
Do we need to specify that the edits and activation must be in the same edit session?

Reply:
Yes, I think I made that explicit elsewhere and can be sure to do so here as well :+1:

Reply:
Two Qs:
1. What are the consequences? losing the changes made while editing?
2. This doesn't apply to the create-from-scratch situation below, right? (since they are making a YAML file and then running `oc create`)

Reply:
If you lose the changes, you might activate an incorrect CPMS.
If you save and don't activate, the CPMS generator will overwrite it with what it thinks the desired spec should be.

For create from scratch the generator won't interfere as it won't exist, so yep, all good